React/tic-tac-toe/src/redux/login/actions.js

```diff
@@ -0,0 +1,22 @@
+import { stringArrayToObject } from '../../utils/stringUtils';
+
+export const actions = stringArrayToObject(
+  [
+    'GET_LOGGED_USER',
+    'LOGIN_REQUEST',
```

Comment:
you are not using the LOGIN_REQUEST and the LOGIN_FAILURE.. so, delete them or add a todo